libsrc/c64/soft80_conio.s

```diff
@@ -51,13 +51,9 @@ soft80_init:
         jmp     soft80_kclrscr
 
 soft80_shutdown:
-        lda     #$1b
-        sta     VIC_CTRL1
-        lda     #$03
-        sta     CIA2_PRA
-        lda     #$15
-        sta     VIC_VIDEO_ADR
-        rts
+
+        jsr     $fda3   ; Initialise I/O
+        jmp     $ff5b   ; Initialize screen editor
```

Comment:
I followed your suggestion with https://github.com/cc65/cc65/commit/377f31d08527060d9e2ef6efff6e4c7a058f5dd4 and it works like a charm :-)

Thanks for looking into this issue !